### DISCUSSION TITLE
connection fails: Failed to execute 'postMessage' on 'Worker': [object Array] could not be cloned.

### DISCUSSION THREAD

**Discussion by yiyanwei** [electerm/electerm#2829] [General]
macOS，新建连接， 测试连接，提示这个错误 “connection fails: Failed to execute 'postMessage' on 'Worker': [object Array] could not be cloned.”

**Comment by AnDonger** [comment]
不使用软件内更新功能，通过自助下载(https://github.com/electerm/electerm/releases) , 安装覆盖应该可以解决你的问题
